QueryEngine/RelAlgTranslator.cpp

```diff
@@ -299,7 +299,7 @@ std::shared_ptr<Analyzer::Expr> RelAlgTranslator::translateInput(const RexInput*
       CHECK_EQ("rowid", cd->columnName);
       col_ti.set_size(8);
     }
-    if (rte_idx > 0 && join_type_ == JoinType::LEFT) {
+    if (rte_idx > 0 && join_types_[rte_idx - 1] == JoinType::LEFT) {
```

Comment:
ASan seems to be failing here (and I'd assume at line 312 below).
[asan.txt](https://github.com/mapd/mapd-core/files/1644899/asan.txt)

Reply:
Thanks, I'm looking into it right now.